FogCreek/Glitch-Community
# Charmed albertonykus

## Links
* https://charmed-albertonykus.glitch.me/settings
* https://app.clubhouse.io/glitch/story/3221/add-a-delete-account-section-to-the-new-settings-page-with-a-button-to-delete-the-account
* https://docs.google.com/document/d/1ya5WQraU0OPXf22uCsTGhaNnFU-3XyVLHSqGHzMkTPk/edit#heading=h.3im6lap0txtw

## GIF/Screenshots:

## Changes:
* Adds a button to delete your account, hidden behind a dev toggle

## How To Test:
* It doesn't actually _do_ anything yet, so...just make sure its a button? And it only shows up with the dev toggle on, when you're already logged in?

## Feedback I'm looking for:
Should this be broken out into a separate file and imported?

## Things left to do before deploying:
- [x] Have Tiff add a coffin image to shared-components
- [x] Update/publish shared-components to be able to source the coffin icon
